src/OpenTelemetry.Instrumentation.EntityFrameworkCore/README.md

```diff
@@ -140,6 +140,16 @@ services.AddOpenTelemetry()
         .AddConsoleExporter());
 ```
 
+### SetDbStatementForText
+
+`SetDbStatementForText` controls whether the `db.statement` attribute is
+emitted. The behavior of `SetDbStatementForText` depends on the runtime used,
+see below for more details.
+
+Query text may contain sensitive data, so when `SetDbStatementForText` is
+enabled the raw query text is sanitized by automatically replacing literal
+values with a `?` character.
+
```

Comment:
Didn't mean to check this in - will fix when I commit the intended README updates shortly.